test/games/strategy/engine/data/annotations/InvalidFieldNameExample.java

```diff
@@ -30,7 +30,5 @@ public void setAttribute(final String attribute) {
   public void resetAttribute() {}
 
   @Override
-  public void validate(final GameData data) throws GameParseException {
-    // TODO Auto-generated method stub
-  }
+  public void validate(final GameData data) throws GameParseException {}
```

Comment:
Bit of a code smell with so many empty implementations of the same method.. Not sure what to be done, but having this much boiler plate is not good.